phpBB/styles/prosilver/template/posting_buttons.html

```diff
@@ -105,6 +105,16 @@
 	<button type="button" class="button button-icon-only bbcode-color" name="bbpalette" id="bbpalette" value="{L_FONT_COLOR}" onclick="change_palette();" title="{L_BBCODE_S_HELP}">
 		<i class="icon fa-tint fa-fw" aria-hidden="true"></i>
 	</button>
+	<!-- IF S_SMILIES_ALLOWED -->
+	<button type="button" class="button button-icon-only smiley_button" accesskey="y" name="addsmiley" title="{L_SMILEYS}">
```

Comment:
something else has to be causing this otherwise why does it not happen on the other dropdowns